### PR TITLE
create category

### PR DESCRIPTION
**Pull request by phuongbk**

### CODE REVIEW

File: app/views/admin/categories/_category.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<aside>
+  <section class="user_info">
+    <h1>
+      <%= category.name %>
+    </h1>
+    <p>
+      <%= category.description %>
+    </p>
+    <p>
+      Create at:<%= category.created_at %>
```
Comment (VoTaiTri) [inline app/views/admin/categories/_category.html.erb]: I18n please

File: app/views/admin/categories/_new_category.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<%= form_for [:admin, @category] do |f| %>
```
Comment (VoTaiTri) [inline app/views/admin/categories/_new_category.html.erb]: rename this file, try "_form.html.erb"

File: app/views/admin/categories/_category.html.erb (hunk 1 above)
Comment (phuongbk) [inline app/views/admin/categories/_category.html.erb]: tell me why?

File: app/views/admin/categories/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<%= form_for [:admin, @category] do |f| %>
+  <%= render "shared/error_messages", object: f.object %>
+
+  <%= f.label :name %>
+  <%= f.text_field :name, class: "form-control" %>
+
+  <%= f.label :Description %>
```
Comment (VoTaiTri) [inline app/views/admin/categories/_form.html.erb]: instead 
```ruby
:Description
```
by 
```ruby
:description
```
or
```ruby
"Description"
```

File: app/views/admin/categories/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<% provide :title, t("title.all_categories") %>
+<h1><%= t "title.all_categories" %></h1>
+<%= link_to t("title.add_category"), new_admin_category_path %>
+<%= will_paginate %>
+  <ul class="users">
+    <%= render @categories %>
+  </ul>
+<%= will_paginate %>
```
Comment (leminhtuan2015) [inline app/views/admin/categories/index.html.erb]: duplicate

**Comment by VoTaiTri** [comment]
LGTM

**Comment by leminhtuan2015** [comment]
LGTM, well done.

File: app/views/admin/categories/_category.html.erb [hunk 5]
```diff
@@ -0,0 +1,16 @@
+<aside>
+  <section class="user_info">
+    <h1>
+      <%= category.name %>
+    </h1>
+    <p>
+      <%= category.description %>
+    </p>
+    <p>
+      <%= t("info.created_time") %><%= category.created_at %>
```
Comment (phonghenry14) [inline app/views/admin/categories/_category.html.erb]: bỏ ()

File: app/views/admin/categories/_category.html.erb [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    </p>
+    <p>
+      <%= t("info.updated_time") %><%= category.updated_at %>
```
Comment (phonghenry14) [inline app/views/admin/categories/_category.html.erb]: bỏ ()

**Pull request merged**